src/components/common/YourCompany/FolioInformation.vue

```diff
@@ -69,8 +73,7 @@ export default class FolioInformation extends Mixins(CommonMixin) {
   /** Whether to hide the component's actions. */
   get hideActions (): boolean {
     // hide actions in a correction filing
-    // hide actions from staff users
-    return (this.isCorrectionFiling || this.isRoleStaff)
```

Comment:
"staff" (ie, an account that can use Staff Payment options) don't see THIS component, so this check is redundant.